pom.xml

```diff
@@ -125,8 +125,8 @@
     <maven-bundle-plugin.version>2.5.3</maven-bundle-plugin.version>
     <os-maven-plugin.version>1.6.2</os-maven-plugin.version>
 
-    <karaf-maven-plugin.version>4.2.6</karaf-maven-plugin.version>
-    <hitachi-karaf-maven-plugin.version>4.2.6-R2</hitachi-karaf-maven-plugin.version>
```

Comment:
This was a custom build with fixes HV did to the karaf maven plugin. There were some PRs to include these changes into the "regular" karaf build but I'm unsure if those went through.
@peterrinehart Is the new version still a custom build with the fixes we had previously made but merged with new stuff from the regular karaf repository?

Reply:
Had some time to look better and notice that you are creating a custom build from here https://github.com/pentaho/karaf/tree/hitachi_custom_4.2.15
What would be ideal is that we verify that the changes we made in our custom build are already included in the vanilla karaf and start using the vanilla version. If not yet included, do what is necessary to get there and do it.
@peterrinehart @rmansoor  can we manage to get some cycles in a team to do this? Even if not for the immediate release it would be good to get rid of this tech debt.